locales-pending/fix.ftl

```diff
@@ -27,6 +27,11 @@ fix-flow-data-broker-profiles-view-data-broker-profiles-content = Data brokers m
 fix-flow-data-broker-profiles-view-data-broker-profiles-button-remove-for-me = Remove them for me
 fix-flow-data-broker-profiles-view-data-broker-profiles-button-remove-manually = I’ll remove them manually
 fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites = View your info on these sites
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-trigger-label = More info
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-title = About these profiles
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph1 = Data brokers make up a $240 billion industry of gathering your personal info and selling it to anyone searching for you. They scrape your data from publicly available sources to create profiles that include your name(s), current and previous addresses, family member names, criminal history, and much more.
```

Comment:
I think we've maybe had this conversation a few times already re: strings, but "$240 billion" doesn't mean a lot without a currency. Although maybe less important if we're never releasing outside of en-US locale.
But my $0.02 is to add the clarity of "$240 billion USD" or whatever order those should be in.

Reply:
Isn't `$` the currency? Or is the problem that there are multiple different currencies that use the same sign?

Reply:
TIL: _"The dollar symbol, $, is perhaps the most recognizable in the world, and is used by more than 20 countries around the globe, including the United States, Australia, New Zealand, Canada, and Hong Kong."_
&mdash; via https://www.investopedia.com/terms/c/currency-symbol.asp